### PR TITLE
stock_packaging_calculator improvements

### PR DESCRIPTION
**Pull request by simahawk**
Atomic commits explain changes.

### CODE REVIEW

**Comment by simahawk** [comment]
CC @rousseldenis

File: stock_packaging_calculator/models/product.py [hunk 1]
```diff
@@ -1,51 +1,76 @@
...
-            single_unit = self.uom_id
-            min_unit = (single_unit.factor, single_unit.name)
-        if min_unit:
-            packagings.append(min_unit)
+        packagings = [Packaging(x.id, x.name, x.qty) for x in self.packaging_ids]
+        # Add minimal unit
+        packagings.append(
+            # NOTE: the ID here could clash w/ one of the packaging's.
+            # If you create a mapping based on IDs, keep this in mind.
+            Packaging(self.uom_id.id, self.uom_id.name, self.uom_id.factor)
```
Comment (guewen) [inline stock_packaging_calculator/models/product.py:42]: should we add a "model" key in the dictionary so we know to what model the id refers?

Comment (simahawk) [reply stock_packaging_calculator/models/product.py:42]: I thought about a flag instead (eg: `is_unit=True`) but it's quite easy to get the unit elem: it's always the last one.

**Review by rousseldenis — APPROVED** [review]
LGTM

**Review by guewen — APPROVED** [review]

**Pull request merged**